### PR TITLE
started german locale

### PR DESCRIPTION
**Pull request by cryptix**
Started a basic german locale. Definitely incomplete and has typos but it's something to start with.

Open questions would be how to deal with terms like _Like_, _Thread_,  _Gathering_, _Channel_ that don't have good translations. I'd keep them English.

ps: found that _Reply_ wasn't using l18n yet.

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
+{
+	"$name": "Deutsch (German)",
```
Comment (mmckegg) [inline locales/de.json]: You can add `de: "German"` to the english translation (and others) to handle the "(German)" part automatically depending on the currently selected language

File: overrides/patchcore/message/html/action/reply.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 
+exports.needs = nest({'intl.sync.i18n': 'first'})
+
 exports.gives = nest('message.html.action')
 
 exports.create = (api) => {
+  const i18n = api.intl.sync.i18n
   return nest('message.html.action', function reply (msg) {
-    return h('a', { href: msg.key, anchor: 'reply' }, 'Reply')
+    return h('a', { href: msg.key, anchor: 'reply' }, i18n('Reply'))
```
Comment (mmckegg) [inline overrides/patchcore/message/html/action/reply.js]: Thanks, good catch!

File: locales/de.json (hunk 1 above)
Comment (cryptix) [reply locales/de.json]: Ah, I see! I wonder how that worked.

**Comment by cryptix** [comment]
hmm.. thanks but the collab access somehow didn't take?

> Only those with write access to this repository can merge pull requests.

**Comment by mmckegg** [comment]
@cryptix did you accept the email invite?

**Comment by cryptix** [comment]
sorry about the noise. found it and have the merge button now.

**Comment by cryptix** [comment]
Some of these are a bit lax and clunky but I have to use it a bit to see how it feels.

**Pull request merged**